src/loaders/genome_collection/compute_genome_taxa_count.py

```diff
@@ -148,13 +142,13 @@ def main():
     count_docs, identical_ranks = _create_count_docs(nodes, kbase_collection, load_version)
     rank_doc = _create_rank_docs(kbase_collection, load_version, identical_ranks)
     # Create taxa counts json file
-    count_json = args.output.name if args.output else f'{kbase_collection}_{load_version}_{GENOME_TAXA_COUNT_FILE}'
+    count_json = args.output.name if args.output else f'{kbase_collection}_{load_version}_{names.COLL_TAXA_COUNT}.json'
```

Comment:
Both import files for tax count are .json if I remember correctly.

Reply:
Ranks might be since there's only one entry, but there are multiple json objects in the non-ranks file so it has to be jsonl, right?

Reply:
```
$ head GROW_2023.06_parsed_genome_taxa_counts.json 
{"_key": "0e836058f603016c36aa98f32ac14b24", "coll": "GROW", "load_ver": "2023.06", "rank": "domain", "name": "Bacteria", "count": 2092, "internal_id": null}
{"_key": "3a2385b00771d651bb93367a3d03f097", "coll": "GROW", "load_ver": "2023.06", "rank": "domain", "name": "Archaea", "count": 1, "internal_id": null}
```

Reply:
```
$ head GROW_2023.06_parsed_genome_taxa_counts_rank.json 
{"_key": "9d84c3354c9d93a29311e84342de0867", "coll": "GROW", "load_ver": "2023.06", "ranks": ["domain", "phylum", "class", "order", "family", "genus", "species"]}
```
for ranks I guess it doesn't matter since there's only one line, but we might as well be consistent

Reply:
There are multiple dicts in taxa_counts.json file and it somehow works for arangoimport. Wow

Reply:
It seems arangoimport supports both JSON and JSONL.

Reply:
To be consistent, all converted to be JSONL format now.